### DISCUSSION TITLE
ManageDeclarationPartition válasz

### DISCUSSION THREAD

**Discussion by bmatyi94** [nav-gov-hu/eVAT#61] [Kérdések és válaszok - Gépi interfész] [answered]
Van aki sikeresen kap választ a ManageDeclarationPartition request-re? Nálam nem jön vissza response, timeout, pedig nem nagy a gzip analitika mérete, a teszt analitika ami feltöltésre került ide a github-ra azzal teszteltem.

**Comment by tothj74** [comment]
12.15-én még sikeresen fel tudtam küldeni.

**Reply by renced42** [reply]
Pontosan mi a hibaüzenet? 
RequestID-t tudsz küldeni? Kérlek, hogy az ilyen típusú kérdéseket Issue-ként add fel.

Köszi!

**Reply by bmatyi94** [reply, marked as answer]
Köszönöm, felvettem issue-ba. Egyébként a requestID: c1d4bd7760a947f6a638f205e732ff
